shell/components/nav/Group.vue

```diff
@@ -339,6 +341,7 @@ export default {
         > H6 {
           text-transform: none;
           padding-left: 16px;
+          padding: 8px 0px 8px 16px;
```

Comment:
It looks like `padding-left` is made redundant now.

```suggestion
          padding: 8px 0px 8px 16px;
```